helm/examples/local/README.md

```diff
@@ -17,22 +17,22 @@ Examples of use:
 
   With a host directory of c:\hpccdata
 
-    helm install hpcc-localfile examples/local/hpcc-localfile --set common.hostpath=/run/desktop/mnt/host/c/hpccdata
+    helm install localfile examples/local/hpcc-localfile --set common.hostpath=/run/desktop/mnt/host/c/hpccdata
     helm install mycluster hpcc/ --set global.image.version=latest -f examples/local/values-localfile.yaml
 
 #### Docker desktop (using osx):
 
   With a host directory of /Users/myuser/hpccdata
 
-    helm install localfile examples/local/hpcc-localfile --set common.hostpath=/Users/myuser/hpccdata
+    helm install hpcc-localfile examples/local/hpcc-localfile --set common.hostpath=/Users/myuser/hpccdata
```

Comment:
this is incorrect (it was the one case that was correct before).
This is the instance name, and should be consistent in all the examples, and match the example - it should be "localfile"